docs/guide/polling.md

```diff
@@ -0,0 +1,171 @@
+# Polling
+
+## Poll helper
+
+Polling your server for new information on the current page is common, so Inertia provides a poll helper designed to help reduce the amount of boilerplate code. In addition, the poll helper will automatically stop polling when the page is unmounted.
+
+The only required argument is the polling interval in milliseconds.
+
+:::tabs key:frameworks
+== Vue
+
+```js
+import { usePoll } from '@inertiajs/vue3'
+usePoll(2000)
+```
+
+== React
+
+```js
+import { usePoll } from '@inertiajs/react'
+usePoll(2000)
+```
+
+== Svelte 4|Svelte 5
+
+```js
+import { usePoll } from '@inertiajs/svelte'
+usePoll(2000)
+```
+:::
+
+If you need to pass additional request options to the poll helper, you can pass any of the `router.reload` options as the second parameter.
+
+
+:::tabs key:frameworks
+== Vue
+
+```js
+import { usePoll } from '@inertiajs/vue3'
+
+usePoll(2000, {
+  onStart() {
+      console.log('Polling request started')
+  },
+  onFinish() {
+      console.log('Polling request finished')
+  }
+})
+```
+
+== React
+
+```js
+import { usePoll } from '@inertiajs/react'
+
+usePoll(2000, {
+  onStart() {
+      console.log('Polling request started')
+  },
+  onFinish() {
+      console.log('Polling request finished')
+  }
+})
+```
+
+== Svelte 4|Svelte 5
+
+```js
+import { usePoll } from '@inertiajs/svelte'
+
+usePoll(2000, {
+  onStart() {
+      console.log('Polling request started')
+  },
+  onFinish() {
+      console.log('Polling request finished')
+  }
+})
+```
+:::
+
+If you'd like more control over the polling behavior, the poll helper provides `stop` and  `start` methods that allow you to manually start and stop polling. You can pass the `autoStart: false` option to the poll helper to prevent it from automatically starting polling when the component is mounted.
+
+:::tabs key:frameworks
+== Vue
+
+```vue
+<script setup>
+import { usePoll } from '@inertiajs/vue3'
+const { start, stop } = usePoll(2000, {}, {
+  autoStart: false,
+})
+</script>
+
+<template>
+  <button @click="start">Start polling</button>
+  <button @click="stop">Stop polling</button>
+</template>
+
+```
+
+== React
+
+```jsx
+import { usePoll } from '@inertiajs/react'
+
+export default () => {
+  const { start, stop } = usePoll(2000, {}, {
```

Comment:
```suggestion
  const { start, stop } = usePoll(2000, {
```
As said `If you need to pass additional request options to the poll helper, you can pass any of the router.reload options as the second parameter.`

and in Manual visits : `router.reload(options)`.

No data as a second argument, unlike get/post/put/patch

Reply:
https://github.com/inertiajs/inertia/blame/9106d74495130a3d82d00daec43c4042b81bdb63/packages/react/src/usePoll.ts#L7 I believe this one is fine, but tell me if I'm missing smth 👀

Reply:
You're right, sorry for the inconvenience !